### PR TITLE
!!![v6r12] getDirectoryMetadata was wrongly set in ro_meta_methods

### PR DESCRIPTION
**Pull request by phicharp**
This was causing a lot of failures in LHCb DMS agents :((((

### CODE REVIEW

File: Resources/Catalog/FileCatalog.py [hunk 1]
```diff
@@ -148,6 +148,7 @@ def r_execute( self, *parms, **kws ):
 
       method = getattr( oCatalog, self.call )
       res = method( *parms, **kws )
+      print res
```
Comment (fstagni) [inline Resources/Catalog/FileCatalog.py]: maybe remove this print, or put in log.debug? :)

Comment (phicharp) [inline Resources/Catalog/FileCatalog.py]: yes, I did sorry ;-)

Le 23 févr. 2015 à 10:51, fstagni <notifications@github.com> a écrit :

> In Resources/Catalog/FileCatalog.py:
> 
> > @@ -148,6 +148,7 @@ def r_execute( self, *parms, **kws ):
> >  
> >        method = getattr( oCatalog, self.call )
> >        res = method( *parms, **kws )
> > +      print res
> maybe remove this print, or put in log.debug? :)
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 
Dr Philippe Charpentier
J09210, Physics Department, CERN, CH-1211 Genève 23
LHCb Experiment Distributed Computing Coordinator
Office 2-R-007, Tel : +41 22 767 4244 ,  GSM : +41 76 487 0167
mailto:Philippe.Charpentier@cern.ch  http://cern.ch/Philippe.Charpentier

**Comment by atsareg** [comment]
Just a remark - DataLogging is disabled in v6r13 as obsoleted/not used. So, updating it is not necessary for the moment

**Comment by chaen** [comment]
What do you mean "DataLogging is disabled in v6r13" ? We want it working, and we have a student coming specially for that. And btw, updating was necessary since it was throwing an exception

**Pull request merged**